_docs-sources/intro/overview/intro-to-gruntwork.md

```diff
@@ -15,3 +15,65 @@ Gruntwork products focus on Amazon Web Services (AWS). Support for other public
 ### Gruntwork uses Terraform
 
 All Gruntwork products are built on and fully compatible with [open source Terraform](https://terraform.io). The one exception to this is the [Gruntwork Reference Architecture](https://gruntwork.io/reference-architecture/), which uses [Terragrunt](https://terragrunt.gruntwork.io/) (one of our open source tools) to implement an end-to-end architecture.
+
+### Core modules at our core
```

Comment:
Consider rephrasing. Perhaps "General anatomy of a Gruntwork module"?

Reply:
You don't like my "Core ... core" humor? /sigh

I like your phrasing. Committed in f779c49